modules/auxiliary/gather/apache_superset_priv_esc.rb

```diff
@@ -170,7 +171,7 @@ def run
           status: Metasploit::Model::Login::Status::UNTRIED
         })
       end
-      print_good("Found #{db_type} database #{db_name}: #{db_user}:#{db_pass}@#{db_host}:#{db_port}")
+      print_good("Found #{db_display_name}: #{db_type}://#{db_user}:#{db_pass}@#{db_host}:#{db_port}/#{db_name}")
```

Comment:
This adds the display name to the output which might be handy and formats the rest as a connection string.